angular/projects/public-nrpti/src/app/records/records-list/records-list-resolver.ts

```diff
@@ -25,10 +25,6 @@ export class RecordsListResolver implements Resolve<Observable<object>> {
       schemaList = params.activityType.split(',');
     }
 
-    if (params.act) {
-      params.act = RecordUtils.appendActCodesToActNames(params.act, this.factoryService);
```

Comment:
appendActCodesToActNames was a function written just for this. it is used nowhere else and could be removed as well to avoid bloating the code base.